bootstrap/hook.go

```diff
@@ -64,6 +64,7 @@ func newHookScriptWrapper(hookPath string) (*hookScriptWrapper, error) {
 	if err != nil {
 		return nil, err
 	}
+	defer h.scriptFile.Close()
```

Comment:
Should we be catching this error?

Reply:
I'm not sure! I might merge what we've got and leave that for your perusal later?